tests/models.py

```diff
@@ -289,6 +289,48 @@ def jinja_udf(name):
         result = task.render_template('', "{{ 'world' | hello}}", dict())
         self.assertEqual(result, 'Hello world')
 
+    def test_descendants(self):
```

Comment:
Non happy flow (e.g.): have three dags in the database two of which are dependent on each other and one which is not. Tasks from that third dag should not have been touched when grabbing descendants.